templates/beta-post.adoc

```diff
@@ -165,18 +177,33 @@ Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
 [source,gradle]
 ----
 dependencies {
-    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.RELEASE_VERSION-beta,)'
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.RELEASE_VERSION-beta,)'
 }
 ----
-
+// // // // // // // //
+// Above:
+// Replace RELEASE_VERSION with the version number of Open Liberty
+// // // // // // // //
 Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
 
-To enable the new beta features in your app, add them to your `server.xml`:
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
```

Comment:
Can you move this whole section about server.xml to just before the "Try it now" heading? So that the Try it now section for Jakarta EE 9 looks much like the one in the All Beta Features section.

Reply:
Done